src/test/java/org/apache/maven/plugins/war/AbstractWarMojoTest.java

```diff
@@ -74,7 +74,7 @@ protected void configureMojo( AbstractWarMojo mojo, List<String> filters, File c
             new DefaultMavenExecutionRequest().setSystemProperties( System.getProperties() ).setStartTime( new Date() );
 
         MavenSession mavenSession =
-            new MavenSession( (PlexusContainer) null, (RepositorySystemSession) null, request, null );
+            new MavenSession( null, null, request, null );
```

Comment:
Are you sure this is a good idea? Imagine a new constructor came  up with four other parameters.

Reply:
Yes i can revert it. intellij suggested this change

Reply:
Please revert this unrelated change